docs/docs/internal/governance--how-to-fast-track.md

```diff
@@ -0,0 +1,199 @@
+
+##### Parameters
+- `threshold`: The threshold describes the amount of council members required to accept the proposal.
+- `proposal`: As proposal we use democracy.fastTrack, with the following params:
+    - `proposalHash`: Hash of the extrinsic which will be proposed. The same used during `#1`.
+    - `proposalLengthInVoting`: How many blocks the proposal will stay in referenda.
+    - `proposalDelayUntilVoting`: How many the proposal will be delayed until it goes into referenda.
+- `lengthBound`: a value higher or equal to the length of the SCALE encoded proposal in bytes.[^1]
+
+##### Call example
+```typescript
+const {
+	data: [
+		resultAccount, resultProposalIndex, resultProposalHash, resultThreshold
+	]
+} = await sendAndWaitForSuccess(
+	api,
+	technicalCouncilMembers[0],
+	api.events.technicalCommittee.Proposed.is,
+	api.tx.technicalCommittee.propose(
+		threshold,
+		api.tx.democracy.fastTrack(
+			hashOfProposalToPropose, 
+			proposalLengthInVoting, 
+			proposalDelayUntilVoting
+		),
+		lengthBound
+	)
+);
+```
+
+
+#### 5. Technical committee votes on proposal
+Now the members of the technical committee can vote on the newly created internal proposal.
+
+![e](./governance--how-to-fast-track--images/5-technical-committee--proposal-voting.png)
+
+##### Parameters
+- `technicalCommitteeProposalHash`: Hash of the technical committee proposal. Received during `#4` as `resultProposalHash`.
+- `technicalCommitteeProposalIndex`: Index number of the technical committee proposal. Received during `#4` as `resultProposalIndex`.
+- `voteChoice`: true / false depending on the desired outcome of the technical committee proposal.
+
+##### Call example
+```typescript
+api.tx.technicalCommittee.vote(
+	technicalCommitteeProposalHash,
+	technicalCommitteeProposalIndex,
+	true
+)
+```
+
+
+#### 6. Technical committee closes the proposal
+As soon as the voting threshold has been reached, the technical committee can close the proposal & get it into effect.
+
+![f](./governance--how-to-fast-track--images/6-technical-commitee--proposal-closing.png)
+
+##### Parameters
+- `technicalCommitteeProposalHash`: Hash of the technical committee proposal. Received during #4 as "resultProposalHash".
+- `technicalCommitteeProposalIndex`: Index number of the technical committee proposal. Received during #4 as "resultProposalIndex".
+- `weightBound`: the maximum amount of Substrate weight the proposal can use. If the proposal call uses more, the call will revert.[^1]
+- `lengthBound`: a value higher or equal to the length of the SCALE encoded proposal in bytes.[^1]
+
+##### Call example
+```typescript
+api.tx.technicalCommittee.close(
+	technicalCommitteeProposalHash, 
+	technicalCommitteeProposalIndex, 
+	weightBound, 
+	lengthBound
+)
+```
+
+
+#### Fast Tracking was successful
+Now the desired proposal will be in voting & all users can vote for their desired outcome.
+This will be the case until it has reached the defined `proposalLengthInVoting` during `#4`.
```

Comment:
This variable overrides the one for a normal proposal in referenda? Please mention this